config/release-cycle.php

```diff
@@ -5,10 +5,22 @@
     '3.x' => [
         'php' => '>=8.0.0'
     ],
-    '2.0' => [
+    '2.2.x ¹' => [
+        'php' => '>=8.1.0'
+    ],
+    '>= 2.0.50' => [
+        'release' => '2024-05-30',
+        'enhancements' => '2024-05-30',
+        'bugfixes' => '2026-11-23',
+        'eol' => '2027-11-23',
+        'php' => '>= 7.3, <= 8.4'
+    ],
+    '<= 2.0.49.x' => [
         'release' => '2014-10-12',
-        'enhancements' => '2018-02-19',
-        'php' => '>=5.4.0'
+        'enhancements' => '2023-08-29',
+        'bugfixes' => '2023-10-31',
+        'eol' => '2026-11-23',
+        'php' => '>= 5.4, <= 8.3'
```

Comment:
That is extra maintenance burden. I'd avoid that by always requesting to update 2.0.x.

Reply:
@cebe said he'll do it.